lib/appsignal/plug.ex

```diff
@@ -21,13 +21,16 @@ if Appsignal.plug?() do
               conn
               |> Plug.Conn.put_private(:appsignal_transaction, transaction)
               |> Plug.Conn.register_before_send(fn conn ->
-                Appsignal.Plug.finish_with_conn(transaction, conn)
+                Appsignal.Plug.try_set_action(transaction, conn)
+                conn
               end)
 
             try do
               super(conn, opts)
             catch
               kind, reason -> Appsignal.Plug.handle_error(conn, kind, reason, System.stacktrace())
```

Comment:
Does the unhandled exit not go through here but ends up in the error logger instead? We'd still receive it, but without the action name.

This part:
https://github.com/appsignal/appsignal-elixir/blob/cc2f46fce1220fbc00d7f002ea62059792952bd0/lib/appsignal/error_handler.ex#L30-L34